src/main/java/org/folio/search/utils/SearchUtils.java

```diff
@@ -157,4 +158,8 @@ public static String getEffectiveCallNumber(String prefix, String callNumber, St
   public static <T> Stream<T> toSafeStream(List<T> nullableList) {
     return CollectionUtils.isNotEmpty(nullableList) ? nullableList.stream() : Stream.empty();
   }
+
+  private static String getFolioEnvAndMakeValidEsIdentifier() {
+    return getFolioEnvName().replace("-", "_").toLowerCase();
+  }
```

Comment:
Is "-" not allowed? The doc did not mention that explicitly https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-indexing.html

Reply:
You're right, removed this method.